dependencies.gradle

```diff
@@ -11,7 +11,7 @@ ext {
     androidxTestVersion = "1.4.0"
     realmVersion = "10.7.1"
     retrofitVersion = "2.9.0"
-    navVersion = "2.3.5"
+    navVersion = "2.3.0"
```

Comment:
Why did we downgrade? Maybe this was temporary?

Reply:
I don't know for some reason my Ide doesn't support the 2.3.5 version.